source/Ship.cpp

```diff
@@ -797,6 +799,8 @@ void Ship::Save(DataWriter &out) const
 		
 		cargo.Save(out);
 		out.Write("crew", crew);
+		if(Preferences::Has("Rehire extra crew when lost"))
```

Comment:
Let's only write this value out when it is non-zero (and let's write it out regardless of if the Preference was set).